<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timer</class>
 <widget class="QMainWindow" name="Timer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>395</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>395</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>395</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Timer 1.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">vertical-align: center;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>420</width>
      <height>360</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Таймер</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 194, 126);
background-color: rgb(245, 194, 17);
background-color: rgb(222, 221, 218);</string>
      </property>
      <property name="text">
       <string>Начать</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>120</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>56</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Рабочее время</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>146</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Учебное время</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(237, 51, 59);
background-color: rgb(51, 209, 122);
background-color: rgb(222, 221, 218);

</string>
      </property>
      <property name="text">
       <string>Старт/Финиш</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Отдых</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>120</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>118</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>45</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>118</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>3</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 194, 126);
background-color: rgb(245, 194, 17);
background-color: rgb(222, 221, 218);</string>
      </property>
      <property name="text">
       <string>Начать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(46, 194, 126);
background-color: rgb(245, 194, 17);
background-color: rgb(222, 221, 218);</string>
      </property>
      <property name="text">
       <string>Начать</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>280</y>
        <width>120</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>118</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>5</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>За 5 дней</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Рабочее время</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>225</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Отдых</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_6">
      <property name="geometry">
       <rect>
        <x>167</x>
        <y>270</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>125</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Учебное время</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_4">
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>170</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_5">
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>70</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>За 21 день</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Рабочее время</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_7">
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>170</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_8">
      <property name="geometry">
       <rect>
        <x>167</x>
        <y>270</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>125</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Учебное время</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_9">
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>70</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>225</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Отдых</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
